include/forms.php

```diff
@@ -2,12 +2,11 @@
 /**
  * Library for doing repetetive forms stuff
  */
-function make_select($name, $selected, $ar, $jscallback="", $cssclass=''){
```

Comment:
This change breaks other parts of the website. Can you describe the need for it?